packages/mdc-ripple/_mixins.scss

```diff
@@ -169,6 +168,41 @@ $mdc-ripple-common-styles-emitted_: false !default;
   @include mdc-states-hover-opacity(map-get($opacity-map, "hover"));
   @include mdc-states-focus-opacity(map-get($opacity-map, "focus"), $has-nested-focusable-element);
   @include mdc-states-press-opacity(map-get($opacity-map, "press"));
+
+  @if $activated-color {
```

Comment:
Can you create a `mdc-activated-state($color, $has-nested-focusable-element)` mixin which just handles activated state? It seems like its isolated logic